File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F11.P2.go
package main

import "fmt"

func subRange(xs []int) int {
	// BEGIN PROBE F11.P2
	part := xs[1:4]
	return part[0]
	// END PROBE F11.P2
}

func main() {
	fmt.Println(subRange([]int{10, 20, 30, 40, 50}))
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F11.P2.kt
fun subRange(xs: MutableList<Int>): Int {
    // BEGIN PROBE F11.P2
    val part = xs.subList(1, 4)
    return part[0]
    // END PROBE F11.P2
}

fun main() {
    println(subRange(mutableListOf(10, 20, 30, 40, 50)))
}
